critiquebrainz/frontend/views/review.py

```diff
@@ -198,6 +198,10 @@ def create():
         return redirect(url_for('search.selector', next=url_for('.create')))
     return render_template('review/write.html', form=form, entity_type=entity_type, entity=entity)
 
+    if review.entity_type == 'release_group':
```

Comment:
This part of code will never be reached because there's a `return` right above it.